docs/blitz/design/advanced_actions/advanced.rst

```diff
@@ -555,11 +559,100 @@ Examples are provided below for these conditional statements.
                              command: show version
                              device: uut
                          - sleep:
                              sleep_time: 1
 
+
+**Example-6: Running actions with if else condition**
+
+.. code-block:: YAML
+
+    test:
+        source:
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+        devices: ['uut']
+        test_sections:
+            - apply_config:
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output                 # the 1500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet1
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output_1                 # the 2500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet10
+                - run_condition:
+                  - if: "%VARIABLES{gims_output} == 1200"   # gims_output output is 1500, if condition fails
+                    actions:
+                      - parse:
+                          command: show version
+                          device: uut
+                      - sleep:
+                          sleep_time: 1
+                  - else:                                 # actions under else is executed here
+                    actions:
+                      - parse:
+                          command: show version
+                          device: uut
+
+
+**Example-7: Running actions with if, elif and else condition**
+
+.. code-block:: YAML
+
+    test:
+        source:
+            pkg: genie.libs.sdk
+            class: triggers.blitz.blitz.Blitz
+        devices: ['uut']
+        test_sections:
+            - apply_config:
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output                 # the 1500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet1
+                - api:                                              # api output is equal to 1500
+                     device: uut
+                     function: get_interface_mtu_size
+                     save:
+                       - variable_name: gims_output_1                 # the 2500 is stored in gims_output
+                     arguments:
+                       interface: GigabitEthernet10

```

Comment:
not used again? please remove or please update example to use this

Reply:
updated.